docs/concepts/why-temporal.md

```diff
@@ -6,26 +6,16 @@ tags:
   - explanation
 ---
 
-import CenteredImage from "../components/CenteredImage.js"
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-<!-- prettier-ignore -->
-import * as WhatIsAWorkflowExecution from '../concepts/what-is-a-workflow-execution.md'
-import * as WhatIsASignal from '../concepts/what-is-a-signal.md'
-import * as WhatIsAQuery from '../concepts/what-is-a-query.md'
-
 One of the aspects of the Temporal System is that it abstracts the complexity of a distributed system.
 Distributed systems exist to scale computation across multiple machines as the potential load of a system changes.
 In theory, a distributed system facilitates a reliable and highly performant application.
 
 However any failure that leaves the downstream part of the application waiting for a response can make things very complicated, especially at a large scale.
 
-<CenteredImage
-imagePath="/diagrams/basic-distributed-system.svg"
-imageSize="75"
-title="Distributed application failures"
-/>
+![Distributed application failures](/diagrams/basic-distributed-system.svg)
```

Comment:
The diagrams don't fare well in dark mode, but that doesn't materially affect this specific PR.

Reply:
updated diagrams!